src/urbit/app/chess.hoon

```diff
@@ -257,6 +257,43 @@
             ::  record that draw offer is gone
             games  (~(put by games) game-id.action u.game(got-draw-offer |))
           ==
+        %resign
+          =/  game-state  (~(get by games) game-id.action)
+          ::
+          ::  we received a resignation with a game id,
+          ::  do we have a game with that id?
+          ?~  game-state
+            ::  if not, error!
```

Comment:
As mentioned in the chat, I have a note to address the "old-style" comments in this PR in line with your previous reviews.